packages/element-plus/make-installer.ts

```diff
@@ -20,7 +19,7 @@ export const makeInstaller = (components: Plugin[] = []) => {
       () => unref(options).zIndex,
       () => {
         const zIndex = unref(options).zIndex
-        if (isNumber(zIndex)) PopupManager.globalInitialZIndex = zIndex
+        if (isNumber(zIndex)) usePopupManager().setGlobalInitialZIndex(zIndex)
```

Comment:
I don't think that this will work with nested popup manager since the variable `globalInitialZIndex` gets initialized every time when you call `usePopupManager`, which means it is not the same variable that you have changed here, would you mind add a unit test covering this?